reoptjl/views.py

```diff
@@ -2362,4 +2363,221 @@ def get_bau_column(col):
 
 ##############################################################################################################################
 ################################################### END Results Table #########################################################
+##############################################################################################################################
+
+##############################################################################################################################
+################################################# START Hourly Rate Table #####################################################
+##############################################################################################################################
+
+def hourly_rate_table(request: Any) -> HttpResponse:
+    """
+    Generate an Excel file with hourly rate data for one or more scenarios.
+    Accepts multiple run_uuid values via GET request parameters.
+    
+    Format:
+    - Column 1: DateTime (based on first run_uuid's year and time_steps_per_hour)
```

Comment:
The format of the data depends on the custom table config, right?

Reply:
I see now that we don't really have a template where we can specify which timeseries data we want to include in the spreadsheet, like we do for the /get_results_table endpoint with the input custom_table_name parameter. Ideally we could generalize this endpoint so that we're not just creating a specific table that the ANCCR project wants, and instead it could be used to generate other tables with TBD configuration. Can you give that a try? Ideally we'd have a separate file where we specify that, like we have for custom_table_config.py, but I'd also be fine if we wanted to include that in the same "helper.py" file you created for this endpoint.